plugins/sge.py

```diff
@@ -232,7 +234,7 @@ def _get_host_qname_np(self, queue_elem, anonymize):
         else:
             # out of the 3, np information is the most likely to be missing from a node
             worker_node['np'] = 0
-
+        # import wdb; wdb.set_trace()
```

Comment:
better leave a TODO tag here, because  the hash signals some possible future work!